tests/test_time_travel.py

```diff
@@ -0,0 +1,136 @@
+from openai import OpenAI, AsyncOpenAI
+import openai
+from openai.resources.chat import completions
+import agentops
+from dotenv import load_dotenv
+
+load_dotenv()
+client = OpenAI()
+
+agentops.init(tags=["TTD Test", openai.__version__])
+
+try:
+    chat_completion_1 = client.chat.completions.create(
+        messages=[
+            {
+                "content": "Come up with a random superpower that isn't time travel. Just return the superpower in the format: 'Superpower: [superpower]'",
+                "role": "user",
+            }
+        ],
+        model="gpt-3.5-turbo-0125",
```

Comment:
This isn't really a test by pytest standards. Can we move it to the `core_manual_tests` tests folder and change the file name to not start with `test_` as tox and pytest will try to automatically run these files?

Reply:
Yes

Reply:
agreed ^